tracpro/static/less/trac.less

```diff
@@ -203,4 +203,44 @@ footer {
 /* Override datepicker colors */
 .ui-datepicker select.ui-datepicker-month, .ui-datepicker select.ui-datepicker-year {
   color: #000000;
```

Comment:
When I narrow my window, the circles and text end up scrunched on top of each other while there's still horizontal space that they could spread out in.

![image](https://cloud.githubusercontent.com/assets/2788679/26582128/e54614f2-450d-11e7-8e0c-ef5e9b7d5e3a.png)